bitnami/phpmyadmin/values.yaml

```
@@ -473,10 +553,37 @@ metrics:
     annotations:
       prometheus.io/scrape: "true"
       prometheus.io/port: "{{ .Values.metrics.service.port }}"
+    ## @param service.clusterIP phpmyadmin service Cluster IP
```

```suggestion
    ## @param metrics.service.clusterIP phpmyadmin service Cluster IP
```
And the parameters below also need the `metrics.` at the begining